lib/bounces.js

```diff
@@ -47,6 +47,19 @@ module.exports = function (log, error) {
         // But we can measure it!
         log.increment('account.email_bounced.already_verified')
       }
+      // Because we have no session token on this endpoint, metrics context
+      // metadata was stashed against a synthesized token for the benefit of
+      // the activity events. This fake request object allows the correct
+      // metadata to be gathered when we emit the events.
+      var fakeRequestObject = {
+        auth: {
+          credentials: {
+            uid: record.uid.toString('hex')
+          }
+        }
+      }
+
+      log.flowEvent('account.email_bounced', fakeRequestObject)
```

Comment:
I am a little confused on how the email bouncing is working. Why would this be logged here rather than `handleBounce`?